packages/libzip.rb

```diff
@@ -0,0 +1,47 @@
+require 'package'
+
+class Libzip < Package
+  description 'libzip is a C library for reading, creating, and modifying zip archives.'
+  homepage 'https://libzip.org/'
+  version '1.5.1'
+  source_url 'https://libzip.org/download/libzip-1.5.1.tar.xz'
+  source_sha256 '04ea35b6956c7b3453f1ed3f3fe40e3ddae1f43931089124579e8384e79ed372'
+
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libzip-1.5.1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libzip-1.5.1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libzip-1.5.1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libzip-1.5.1-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: 'd219497c9fb7451c26c93dd9379808fbbd936317e765be63beafb8d91db2a84d',
+     armv7l: 'd219497c9fb7451c26c93dd9379808fbbd936317e765be63beafb8d91db2a84d',
+       i686: 'd286172ecf333bed71e0bd1c7713cbbefb5a5434e2592ee5e67cd881bccb5784',
+     x86_64: '4d1fa9fad684b7e4aa92885e247e963ac7ad1a1c85fd7a35cec728491a2e72cc',
+  })
+
+  depends_on 'bz2'
+
+  def self.build
+    Dir.mkdir 'build'
+    Dir.chdir 'build' do
+      system "cmake \
```

Comment:
Can you use `",` instead of backslashes? Same for all of these below, please.

Reply:
It makes for an unpleasant error when you use backslashes.
Example:
```
'foo' failed to install: `really                     long                   configure                         command` exited with 1
```

Reply:
So here's the thing.  I actually prefer this method to the commas because if you ever need to set some environment variables, this is the most readable way to do it.  Granted this package doesn't need any environment variables but sometimes cmake requires them and using commas doesn't work.

Reply:
Just use `env`. It's a lot prettier, and there is no backslashing required.

Reply:
The unpleasant error output with lots of spaces is a fair argument for not using this method.  Why didn't you suggest the commas with the php packages?

Reply:
It's fine, it can get fixed later.  Not a big deal.

Reply:
I did, though I imagine it didn't come across that way.